fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -26,7 +26,7 @@
 
     <!-- Action -->
     <!-- Primary button, Snackbar, Floating action button, Chip selected -->
-    <color name="fx_mobile_action_color_primary">@color/photonViolet60</color>
+    <color name="fx_mobile_action_color_primary">@color/photonViolet50</color>
```

Comment:
App resources - colors / strings etc should not be changed without first confirming with UX.
Given the importance of this particular two colors - used in many places all throughout the app I suggest dropping the color changes to unlock landing other parts of the PR.